src/metamask/index.ts

```diff
@@ -49,6 +51,10 @@ export const getMetaMask = (page: Page): Promise<Dappeteer> => {
         deleteAccount: deleteAccount(page),
         deleteNetwork: deleteNetwork(page),
       },
+      snaps: {
+        acceptDialog: acceptDialog(page),
```

Comment:
could we move invokeSnap here as well?

Reply:
@mpetrunic and installSnap probably as well